Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
```

Comment:
and as for the name, I like that put indent in there. I'm playing with Hypertext now and I think swapping the order of spaces/indent makes more sense. Right? Because first you get all the spaces that this `Renderable` is inheriting from the formatting of previous parents/siblings, and then you see what the requested indentation is that can be added.

Something like 

```swift
...

  func render() -> String
  func render(startingWithSpacesCount: Int, indentingWithSpacesCount: Int) -> String

...
```
We can remove the default for indentingWithSpacesCount. What do you think? To me, we know little about what cases somebody would want to generate indented HTML in and so we can leave that explicit the first you call it. Also, this will keep the autocomplete options users see tidy.